packages/specification/scripts/extractSchemas.ts

```diff
@@ -0,0 +1,83 @@
+import fs from "fs"
+import path from "path"
+import {
+  responseBundleSchema as ResponseBundle,
+  outcomeSchema as OperationOutcome,
+} from "prescriptionSearch"
+import {JSONSchema} from "json-schema-to-ts"
+
+// Record to store schemas
+const schemas: Record<string, JSONSchema> = {ResponseBundle, OperationOutcome}
+
+// Define the folder paths
+const schemasFolder = path.join(".", "schemas")
+const resourcesFolder = path.join(schemasFolder, "resources")
+
+// Create `schemas` folder if it doesn't exist
+if (!fs.existsSync(schemasFolder)) {
+  fs.mkdirSync(schemasFolder)
+}
+
+// Create `resources` folder if it doesn't exist
+if (!fs.existsSync(resourcesFolder)) {
+  fs.mkdirSync(resourcesFolder)
+}
+
+// Function to check if the schema is not an array
+function isNotJSONSchemaArray(schema: JSONSchema | ReadonlyArray<JSONSchema>): schema is JSONSchema {
+  return !Array.isArray(schema)
+}
+
+// Function to collapse examples
+function collapseExamples(schema: JSONSchema): JSONSchema {
+  if (typeof schema !== "object" || schema === null) {
+    return schema
+  }
+
+  // Create a new object for the result
+  const result: JSONSchema = {...schema}
```

Comment:
just use explicit any here and override the eslint rule rather than casting every time we use it.

Reply:
A new object for the result is created using explicit `any`.
```
// eslint-disable-next-line @typescript-eslint/no-explicit-any
const result: any = {...schema}
```